src/routes/Details.tsx

```diff
@@ -1,14 +1,208 @@
 import { useParams } from "react-router-dom";
+import { LineChart, AreaChart } from 'react-chartkick';
+import 'chartkick/chart.js'
+import djs from "dayjs";
+import { currencyFormat, fetchGraphData } from "../utilities";
+import 'chartkick/chart.js'
+import { useQuery } from "@apollo/client";
+import { usEquities, forex, crypto } from "../queries";
+import { useState } from "react";
+import { TimePeriodFilter } from '../components';
 
 export default function Details() {
   let params = useParams();
-  return (
-    <>
-      <main>
-        <div className="flex bg-red-600 grow container mx-auto mt-16 flex-col w-full py-12 items-center">
-          Ticker {params.tickerId}
+  const equitySymbol = params.tickerId;
+  const streamType = params.streamType;
+
+  const [aggLimit, setAggLimit] = useState<number>(30);
+  const [aggPeriod, setAggPeriod] = useState<string>("DAY");
```

Comment:
Could be worth using a union type instead of a string, e.g.

```suggestion
  const [aggPeriod, setAggPeriod] = useState<"DAY" | "MINUTE" | "HOUR">("DAY");
```

Up to you though, not a blocker